android/app/src/main/res/values/styles.xml

```diff
@@ -8,6 +8,7 @@
         <item name="android:forceDarkAllowed">true</item>
         <item name="android:windowTranslucentStatus">false</item>
         <item name="android:statusBarColor">?attr/backgroundColor</item>
+        <item name="android:navigationBarColor">?attr/backgroundColor</item>
```

Comment:
I think this match only the system theme, and will not follow the preferred one in the app. We need something like we do with the status-bar, to control it from the app. Also now the bg and buttons are both white on my phone
![photo_2020-06-18 13 23 06](https://user-images.githubusercontent.com/5794620/85009285-e2ea2e00-b166-11ea-9bc1-a0ae75e9a067.jpeg)
![photo_2020-06-18 13 23 07](https://user-images.githubusercontent.com/5794620/85009289-e4b3f180-b166-11ea-9f96-53132f63984b.jpeg)

Reply:
> I think this match only the system theme, and will not follow the preferred one in the app.
that's fine. 
> Also now the bg and buttons are both white on my phone
that's not cool :(